.pre-commit-config.yaml

```diff
@@ -115,8 +115,18 @@ repos:
           - license-templates/LICENSE.txt
           - --fuzzy-match-generates-todo
       - id: insert-license
-        name: Add license for all YAML files
+        name: Add license for all Helm template files
         exclude: ^\.github/.*$|^.*/.*_vendor/
```

Comment:
You should likely remove the `exclude` (not needed any more)  and have those:
```
        types: [yaml]
        files: ^chart/templates/.*\.ya?ml$
```

Reply:
1. I've added (leaved) `exclude`, because it's included to all other entries. Agree that it's not needed, I'll delete it.

2. The idea is to apply license to all files in chart/templates folder, for example `chart\templates\NOTES.txt`